package_Legion2_CorsairLightingProtocolBoards_index.json

```diff
@@ -227,7 +227,58 @@
                         }
                     ],
                     "toolsDependencies": []
-                }
+                },
+				{
+					"name": "Corsair Lighting Protocol Boards - Adafruit SAMD",
+					"architecture": "samd",
+					"version": "0.1.0",
+					"category": "Contributed",
+					"help": {
+						"online": "https://github.com/Legion2/CorsairLightingProtocolBoards"
+					},
+					"boards": [
+						{
+							"name": "CLP Adafruit Trinket M0"
+						}
+					],
+					"toolsDependencies": [
+						{
+							"packager": "adafruit",
+							"name": "arm-none-eabi-gcc",
+							"version": "9-2019q4"
+						},
+						{
+							"packager": "arduino",
+							"name": "bossac",
+							"version": "1.7.0-arduino3"
+						},
+						{
+							"packager": "arduino",
+							"name": "bossac",
+							"version": "1.8.0-48-gb176eee"
+						},
+						{
+							"packager": "arduino",
+							"name": "openocd",
+							"version": "0.10.0-arduino7"
+						},
+						{
+							"packager": "adafruit",
+							"name": "CMSIS",
+							"version": "5.4.0"
+						},
+						{
+							"packager": "arduino",
+							"name": "CMSIS-Atmel",
+							"version": "1.2.0"
+						},
+						{
+							"packager": "arduino",
+							"name": "arduinoOTA",
+							"version": "1.2.1"
+						}
```

Comment:
when all these tools are required by the original platform, we do not have to list them here. We require the original platform to be installed, because we use its core and libraries.